apis/api-directory/request-right-sizing-api.md

```diff
@@ -0,0 +1,60 @@
+# Container Request Right Sizing Recommendation API
+
+{% swagger method="get" path="savings/requestSizingV2" baseUrl="http://<kubecost-address>/model/" summary="Container Request Right Sizing Recommendation API" %}
+{% swagger-description %}
+The container request right sizing recommendation API provides recommendations for [container resource requests](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) based on configurable parameters and estimates the savings from implementing those recommendations on a per-container, per-controller level. If the cluster-level resources stay static, then there may not be significant savings from applying Kubecost's recommendations until you reduce your cluster resources. Instead, your idle allocation will increase.
+{% endswagger-description %}
+
+{% swagger-parameter in="query" name="algorithmCPU" type="string" required="false" %}
+The algorithm to be used to calculate CPU recommendations based on historical CPU usage data. Options are `max` and `quantile`. Max recommendations are based on the maximum-observed usage in `window`. Quantile recommendations are based on a quantile of observed usage in `window` (requires the `qCPU` parameter to set the desired quantile). Defaults to `max`. To use the `quantile` algorithm, the [ContainerStats Pipeline](/architecture/containerstats-pipeline.md) must be enabled.

```

Comment:
@mbolt35 Is the user still responsible for enabling the ContainerStats Pipeline if applicable? Otherwise I'll omit this line

Reply:
ContainerStats pipeline is enabled by default (can't be disabled) in KC Cloud